charts/dependency-track/templates/api-server/deployment.yaml

```diff
@@ -10,9 +10,15 @@ spec:
   replicas: 1
   selector:
     matchLabels: {{- include "dependencytrack.apiServerSelectorLabels" . | nindent 6 }}
+    {{- if .Values.apiServer.extraPodLabels }}
+    {{- toYaml .Values.apiServer.extraPodLabels | nindent 6 }}
+    {{- end }}
```

Comment:
I *think* including the labels in `selector` is not necessary. The matching should still work with only the vanilla labels.

Reply:
removed from both deployment.yaml